standalone/src/recording/index.tsx

```diff
@@ -23,19 +27,19 @@ import {
 } from './action_types';
 import RecordingView from './components/recording_view';
 
-async function fetchProfileImages(profiles: UserProfile[]) {
+async function fetchProfileImages(ids: string[]) {
     const profileImages: {[userID: string]: string} = {};
     const promises = [];
-    for (const profile of profiles) {
+    for (const id of ids) {
         promises.push(
             runWithRetry(() => {
-                return fetch(`${getPluginPath()}/bot/users/${profile.id}/image`, RestClient.getOptions({method: 'get'})).then((res) => {
+                return fetch(`${getPluginPath()}/bot/users/${id}/image`, RestClient.getOptions({method: 'get'})).then((res) => {
```

Comment:
@cpoile The recorder client won't have any user images so it will need to fetch them for all participants. We could optimize in case of a user re-joining but we are talking small things I think.